src/lib/sveltekit-search-params.ts

```diff
@@ -379,7 +379,7 @@ export function queryParam<T = string>(
 	const { subscribe } = derived<[typeof page, typeof override], T | null>(
 		[page, override],
 		([$page, $override], set) => {
-			if ($override) {
+			if ($override != null) {
```

Comment:
Looks good to me, just small style preference i tend to use `!= undefined` so if you can change that it would be great. I got bit by this soooo many times.

```suggestion
			if ($override != undefined) {
```

Reply:
I know right 😅 It was a nasty one to figure out, but when I changed the default value for this store from false to true I know I was onto something 😂  glad I was able to find it.